libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/AttributeFilterBody.tsx

```diff
@@ -130,9 +137,16 @@ const AttributeFilterBodyCore: React.FC<IAttributeDropdownBodyExtendedProps> = (
     );
 
     const currentWidth = width || 245;
+    const actionsClassnames = cx("gd-dialog-footer dropdown-footer", {
+        "dropdown-footer-mobile": isMobile,
+    });
+    const attributeValuesStyles = {
```

Comment:
We should probably memoize this with useMemo, the isMobile variable is unlikely to change often, so it should help with performance nicely.